### PR TITLE
Hit receptor-controller to get connection status

### PR DESCRIPTION
**Pull request by agrare**
Use the /connection/status endpoint on the receptor controller as the first indication of "available" or "unavailable"

Depends: ~~https://github.com/RedHatInsights/e2e-deploy/pull/1073~~

### CODE REVIEW

**Comment by agrare** [comment]
@gtanzillo e2e-deploy PR is merged, this is ready to go

**Comment by slemrmartin** [comment]
@agrare I made new branch from this one and continuing with 2nd check to `/job`.
This PR doesn't work at this moment

File: lib/topological_inventory/satellite/operations/source.rb [hunk 1]
```diff
@@ -38,19 +38,11 @@ def availability_check
...
-          endpoint_authentications = api_client.list_endpoint_authentications(endpoint.id.to_s).data || []
-          return STATUS_UNAVAILABLE if endpoint_authentications.empty?
-
-          auth_id = endpoint_authentications.first.id
-          auth = Core::AuthenticationRetriever.new(auth_id, identity).process
-          return STATUS_UNAVAILABLE unless auth
-
-          _connection = TopologicalInventory::Satellite::Connection.connection()
-          STATUS_AVAILABLE
+          connection = TopologicalInventory::Satellite::Connection.connection(params["external_tenant"], endpoint.receptor_node)
```
Comment (slemrmartin) [inline lib/topological_inventory/satellite/operations/source.rb:44]: @agrare return `self` in connection method

Comment (agrare) [reply lib/topological_inventory/satellite/operations/source.rb:44]: :+1: fixed

**Pull request merged**